### PR TITLE
fix: scale workloads for helm app created on Devtron

### PR DESCRIPTION
**Pull request by Ashish-devtron**
# Description

When we create a helm app using Devtron chart store using helm or gitops. Scale workload is missing in appDetails.
It is only showing for external helm apps.

Fixes #https://github.com/devtron-labs/devtron/issues/2991

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?
Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [x] created app using helm and gitops and manually hibernate and un-hibernate from dashboard

# Checklist:

* [x] The title of the PR states what changed and the related issues number (used for the release note).
* [ ] Does this PR require documentation updates?
* [ ] I've updated documentation as required by this PR.
* [x] I have performed a self-review of my own code
* [ ] I have commented my code, particularly in hard-to-understand areas

### CODE REVIEW

File: src/components/v2/appDetails/sourceInfo/scaleWorkloads/ScaleWorkloadsModal.component.tsx [hunk 1]
```diff
@@ -233,12 +234,16 @@ export default function ScaleWorkloadsModal({ appId, onClose, history }: ScaleWo
         const _nameSelectionKey = isHibernateReq ? 'scaleDown' : 'restore'
 
         try {
+            let payloadAppId = appId;
```
Comment (shivani170) [inline src/components/v2/appDetails/sourceInfo/scaleWorkloads/ScaleWorkloadsModal.component.tsx:237]: Can directly use appId no need to reassign

File: src/components/v2/appDetails/sourceInfo/scaleWorkloads/ScaleWorkloadsModal.component.tsx [hunk 2]
```diff
@@ -233,12 +234,16 @@ export default function ScaleWorkloadsModal({ appId, onClose, history }: ScaleWo
...
+            if(appDetails.appType != AppType.EXTERNAL_HELM_CHART){
```
Comment (shivani170) [inline src/components/v2/appDetails/sourceInfo/scaleWorkloads/ScaleWorkloadsModal.component.tsx:238]: This is intentional right? We are doing this only for helm & devtron apps

Comment (Ashish-hacker-0) [reply src/components/v2/appDetails/sourceInfo/scaleWorkloads/ScaleWorkloadsModal.component.tsx:238]: yes

File: src/components/v2/appDetails/sourceInfo/scaleWorkloads/ScaleWorkloadsModal.component.tsx [hunk 3]
```diff
@@ -233,6 +234,9 @@ export default function ScaleWorkloadsModal({ appId, onClose, history }: ScaleWo
         const _nameSelectionKey = isHibernateReq ? 'scaleDown' : 'restore'
 
         try {
+            if(appDetails.appType != AppType.EXTERNAL_HELM_CHART){
+                appId=appDetails.clusterId+"|"+appDetails.namespace+"|"+appDetails.appName;
```
Comment (shivani170) [inline src/components/v2/appDetails/sourceInfo/scaleWorkloads/ScaleWorkloadsModal.component.tsx:238]: Please format the file

**Review by shivani170 — APPROVED** [review]

**Review by vivek-devtron — APPROVED** [review]

**Pull request closed without merging**